.travis.yml

```diff
@@ -17,7 +17,8 @@ before_install:
   - "phantomjs --version"
 
 install:
-  - "npm install"
+  - "npm install -g yarn"
+  - "yarn install"
```

Comment:
Please use `yarn install --pure-lockfile --no-emoji` to not produce a lock file if there is none and not output the emojis in the travis log.